Makefile

```
@@ -231,6 +235,12 @@ image:
 endif
 	@if [ "$(DOCKER_PUSH)" = "true" ] ; then docker push $(IMAGE_PREFIX)argocd:$(IMAGE_TAG) ; fi
 
+.PHONY: armimage
+# The "BUILD_ALL_CLIS" argument is to skip building the CLIs for darwin and windows
+# which would take a really long time.
+armimage:
+	docker build -t $(IMAGE_PREFIX)argocd:arm-$(IMAGE_TAG) . --build-arg BUILD_ALL_CLIS="false"
```

```suggestion
	docker build -t $(IMAGE_PREFIX)argocd:$(IMAGE_TAG)-arm . --build-arg BUILD_ALL_CLIS="false"
```

A common practice is to add `-arm` suffix instead of prefix as this allows for easier sorting when looking at tags.